Patch from markusg26@gmx.de to fix building of corba module.
See Gentoo Bug #132776.

Index: vlc-0.8.5/modules/control/corba/corba.c
===================================================================
--- vlc-0.8.5.orig/modules/control/corba/corba.c
+++ vlc-0.8.5/modules/control/corba/corba.c
@@ -28,8 +28,8 @@
 #include "MediaControl.h"
 #include "orbit/poa/portableserver-poa-type.h"
 
-#include <vlc/control.h>
 #include <vlc/vlc.h>
+#include <vlc/mediacontrol.h>
 #include <vlc/intf.h>
 #include <vlc/vout.h>
 #include <vlc/aout.h>
